### ISSUE TITLE
ipywidget value not updated properly in VS Code

### ISSUE THREAD

**Issue by saravanabalagi** [microsoft/vscode-jupyter#8408]
Reopening previously logged bug #7163

## Environment data

<details>
<summary> VSCode About</summary>

```
Version: 1.62.3
Commit: ccbaa2d27e38e5afa3e5c21c1c7bef4657064247
Date: 2021-11-17T08:00:36.721Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.11.0-41-generic snap
```
</details>

-   Jupyter Extension version (available under the Extensions sidebar): v2021.10.1101450599
-   Python Extension version (available under the Extensions sidebar): v2021.11.1422169775
-   OS (Windows | Mac | Linux distro) and version: Ubuntu 20.04
-   Python and/or Anaconda version: 3.8.10
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): poetry venv
-   Jupyter server running: Local

## Expected behaviour

Widget values update (It works in jupyter notebook)

![something1](https://user-images.githubusercontent.com/8567893/144273991-f99d1ce1-3d90-4273-b06e-a586e314bc45.gif)

## Actual behaviour

Widget values do not update

![something](https://user-images.githubusercontent.com/8567893/144273314-1e286145-9ae6-4cec-b41c-3fd592431ccc.gif)

### Cell 1

```py
from ipywidgets import widgets
siteTextBox = widgets.Dropdown(
    options=['site1', 'site2'],
    value='site1',
    description='Site:',
    disabled=False,
)
siteTextBox
```
### Cell 2

```py
siteTextBox.value
```

**Comment by rchiodo** [comment]
Thanks for the bug. This doesn't seem to repro for me (in insiders or stable version of VS code):
![widgetUpdate](https://user-images.githubusercontent.com/19672699/144289083-2e2d9640-5844-4a34-aa45-83648c511155.gif)

Your gif is a little hard to follow. It almost looks like you rerun the first cell?

**Comment by rchiodo** [comment]
Your old bug made it sound like it went away. I wonder if the kernel you're using has something to do with it. 

Can you also run this code and tell me the values:

```
import IPython
print(IPython.__version__)
import ipykernel
print(ipykernel.__version__)
```

Thanks

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/8378
FYI - This has been fixed in insiders.
@rchiodo (you can repro in stable when using non-raw or remote)
